src/pages/index.astro

```diff
@@ -0,0 +1,722 @@
+                    text-decoration: none;
+
+                    @include section-symbol;
+                  }
+                }
+              }
+            }
+
+            .alternatives {
+              display: flex;
+              justify-content: center;
+              gap: 10px;
+              flex-wrap: wrap;
+
+              .alternatives-title {
+                font-size: 1.1rem;
+                font-weight: 300;
+                color: #666;
+                letter-spacing: 0.07em;
+                font-size: 1.4em;
+                text-transform: uppercase;
+                margin: 0;
+                line-height: 1;
+                word-wrap: break-word;
+              }
+
+              .alternatives-list {
+                margin: 0;
+                font-size: 1.1em;
+                list-style: none;
+                padding: 0;
+                display: flex;
+                gap: 10px;
+                flex-wrap: wrap;
+                justify-content: center;
+
+                .alternative-link {
+                  color: inherit;
+                  text-decoration-thickness: 3px;
+                  text-decoration-line: underline;
+                  text-decoration-skip-ink: none;
+                }
+              }
+            }
+
+            &.even {
+              background: #eee;
+
+              .code {
+                background: white;
+              }
+            }
+
+            .engines {
+              display: flex;
+              flex-direction: row;
+              flex-wrap: wrap;
+              justify-content: center;
+              gap: 24px;
+              max-width: 120rem;
+
+              .engine {
+                width: 30rem;
+                min-width: 0;
+
+                .engine-name {
+                  font-weight: 300;
+                  color: #666;
+                  letter-spacing: 0.07em;
+                  font-size: 1.4em;
+                  text-transform: uppercase;
+                  margin-bottom: 0;
+                  margin-top: 8px;
+                  line-height: 1;
+                }
+              }
+            }
+          }
+        }
+      }
+
+      footer {
+        margin: 3em auto;
+        font-size: 1.4em;
+        text-align: center;
+        padding-left: 16px;
+        padding-right: 16px;
+
+        a {
+          color: inherit;
+        }
+      }
+    </style>
+
+    <script>
+      import isEven from '../utils/is-even';
+
+      const searchInput = document.querySelector('#search');
+      const comparisonsParent = document.querySelector('#comparisons');
+
+      function resetResults() {
+        comparisonsParent.classList.remove('no_search_results');
+
+        for (const hiddenCategoryOrComparison of document.querySelectorAll(
+          '.category.hidden, .comparison.hidden'
+        )) {
+          hiddenCategoryOrComparison.classList.remove('hidden');
+        }
+
+        for (const [index, category] of document
```

Comment:
Nit: this can be achieved very easily with a simple CSS selector: `:nth-child(even)` and doesn't have to be maintained here and in the search listener

Reply:
`:nth-child(even)` doesn't respect whether the child is hidden or not. This means we have to manually update this.

Reply:
🤦 I spend too much time writing React code, those elements aren't getting dropped from the DOM. Fair point!